content/blog/2016/branding-type.md

```diff
@@ -170,15 +176,27 @@ method of mixing and matching, but neither did we feel that any one
 element collage encapsulated our brand. We tried style tiles, but those
 felt both too specific and not specific enough.
 
-![image][1]
+{% call embed.img(
+    src='blog/2016/brand/type-styletiles.jpg',
+    alt=''
```

Comment:
Empty alt should be applied by default, without needing to set it here.  (I just confirmed locally, and will push the change)

Reply:
I see `alt` without the `alt=""` and I thought you needed the latter?

Reply:
Where do you see that? When I look at the output code, I see:

```html
<img src="/assets/images/blog/2016/brand/type-styletiles.jpg" alt="" />
```

Reply:
![Screen Shot 2020-03-18 at 12 28 28 PM](https://user-images.githubusercontent.com/1581694/76989286-1ae99700-6914-11ea-9642-fff3ca3f7c6c.jpg)

in chrome on the deploy preview.

Reply:
I see Firefox has the `alt=""` but Safari and Chrome do not. 
![safari](https://user-images.githubusercontent.com/1581694/76989537-859ad280-6914-11ea-95b3-b3f6ea4d5507.jpg)
![chrome](https://user-images.githubusercontent.com/1581694/76989539-86336900-6914-11ea-92a8-5a7517d8aace.jpg)

Reply:
Yeah, the generated source code **does** have it, so I think this is just a difference in how developer tools display the empty tag.